File: src/main/java/com/graqr/threshr/model/redsky/store/PhysicalSpecifications.java
package com.graqr.threshr.model.redsky.store;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record PhysicalSpecifications(@JsonProperty("total_building_area") Long totalBuildingArea,
                                     @JsonProperty("merchandise_level") Long merchandiseLevel,
                                     String format) {}

File: src/main/kotlin/com/graqr/threshr/model/redsky/store/PhysicalSpecifications.kt
package com.graqr.threshr.model.redsky.store

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class PhysicalSpecifications(
    @param:JsonProperty("total_building_area") val totalBuildingArea: Long,
    @param:JsonProperty("merchandise_level") val merchandiseLevel: Long,
    val format: String
) {}
